src/components/Page/metaContent.tsx

```diff
@@ -63,8 +64,10 @@ export default function MetaContent({
           setMenuIsOpen={setMenuIsOpen}
           buttonsRef={buttonsRef}
         ></Menu>
+
         <ContentStyle menuIsOpen={menuIsOpen}>
           <div>
+            <LinkCardCollection />
```

Comment:
LinkCardCollection currently doesn't accept props; how will we supply different links on different pages? It might be easier for re-usability if we make this a little more composable. For example, if our usage looks like this:

```
<LinkCardCollection>
   <LinkCard href="/page-a">Page A</LinkCard>
   <LinkCard href="/page-b" isExternal>Page B</LinkCard>
   <LinkCard href="/page-c">Page C</LinkCard>
</LinkCardCollection>
```

Then it will be easier for us to use on different pages with different links. In this case, your `<LinkCardCollection>` component would not need to use `<Collection />` and could just render 
```
<Flex className="link-card-collection">{children}</Flex>
``` 
or 
```
<Grid className="link-card-collection">{children}</Grid>
```
depending on if you want to use Flex or Grid. 

Additionally, let's make sure we add in some initial responsive styling. These can probably stack on mobile, and then move to 2 in a row, and 4 in a row depending on viewport size and how you want to style them (we'll probably have to adjust once we get the real design).

Reply:
Agreed, changed it to use Grid, thank you!